Classes/Console/Install/Upgrade/UpgradeWizardExecutor.php

```diff
@@ -80,6 +87,8 @@ public function executeWizard(string $identifier, array $rawArguments = [], bool
             $hasPerformed = $upgradeWizard->performUpdate($dbQueries, $message);
         }
 
+        $message = trim($message . PHP_EOL . $output->fetch());
```

Comment:
This overwrites the messages passed as reference to old upgrade wizards.
This line needs to be moved up in the if statement that applies for wizards implementing the new interface.